### ISSUE TITLE
State specific transition freezes the app

### ISSUE THREAD

**Issue by moltar** [shprink/ionic-native-transitions#111]
I have this unique case with resolve doing a state change twice. This is done for deep linking to a specific state, yet retaining (or more correctly creating) history, so that the "back" button still works.

The main thing to look at is the `transition` resolve on the `tab.chat-detail` state.

You can see this:

```js
            $timeout(function() {
              $state.go('tab.chats', {}, goConfig);
            }, 0);

            // now go back to this state
            $timeout(function() {
              $state.go('tab.chat-detail', stateParams, goConfig);
              deferred.resolve(true);
            }, 300);
```

When I add this part:

```js
      nativeTransitions: {
        "type": "flip",
        "direction": "up"
      },
```
The whole app freezes.

Here is the code example repo:

https://github.com/moltar/nativepagetransitions

I did all the changes in stages of commits. If you remove the last commit, everything works.

**Comment by yaoyonstudio** [comment]
got the same problem.

**Comment by TomSeldon** [comment]
I'm having what I think is the same issue, but replicated differently.

If I have a link to a state (using `native-ui-sref` and click the link multiple times in quick succession, then it freezes the app. I can see in the console that multiple native transitions are attempted, and the app *does* go to the correct state in the background, but the screenshot overlayed over the webview is never removed.

**Comment by jawgardner** [comment]
I have this same issue. The console reflects that native transitions are occurring but the screenshots seem to stick around.

**Comment by AlexejKutsko** [comment]
I have same problem. there's solution?

**Comment by TomSeldon** [comment]
Unfortunately, for us the solution was to stop using the plugin as it's no longer maintained. It's a shame, as it worked really nicely with the exception of this bug.

The occasional instance where this bug would completely lock up the app meant it was a no go, and with it no longer being maintained, it just wasn't practical to invest any further time using it, debugging it, or trying to fix it.

I'm hoping that someone picks this project up (I don't have the time) as it really is quite good, but for me, it's back to standard Ionic transitions!

**Comment by ap1969** [comment]
Yep, same issue as jawgardner above: The console reflects that native transitions are occurring but the screenshots seem to stick around.

**Comment by EternallLight** [comment]
Facing the same issues. As a workaround, we could disable native transitions for the parts where they freeze the app, but this is likely not the best solution.
